op-defender/psp_executor/api_test.go

```diff
@@ -0,0 +1,83 @@
+package psp_executor
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+// TestHandlePost tests the handlePost function for various scenarios.
+func TestHandlePost(t *testing.T) {
+	tests := []struct {
+		name           string
+		body           RequestData
+		expectedStatus int
+		expectedBody   string
+	}{
+		// The next tests case are commented for next PRs and don't require review for now.
+
+		// {
```

Comment:
let's remove commented code.

Reply:
I agree we should remove the comment code but I would like to keep the commented code in this one, I will ship the next PR 2 and PR 3 with the commented code quickly after this one.
Do you think this is an issue?